### ISSUE TITLE
some tests in scan_verb_test are void due to faulty logic and assertions

### ISSUE THREAD

**Issue by gkc** [atsign-foundation/at_server#1194]
### Describe the bug

Lines 133, 134: Assertion will never be true because (1) the response will never contain 'update:public' (2) the key will be in lowercase
```dart
    expect(
        false, response.contains('"update:public:verifyingTTL$firstAtsign"'));
```

Additionally, it's not clear how the test is intended to work. These lines 126 to 129 suggest that by the time the scan is issued, the ttl has not yet passed, and so the key SHOULD be returned from a scan
```dart
    //update ttl so that key is expired
    await socket_writer(socketFirstAtsign!,
        'update:ttl:6000:public:verifyingTTL$firstAtsign 1');
    sleep(Duration(seconds: 2));
    await socket_writer(socketFirstAtsign!, 'scan');
```

Lines 156, 157: Assertion will never be true because (1) the response will never contain 'update:public' (2) the key will be in lowercase
```dart
    expect(
        false, response.contains('"update:public:verifyingTTB$firstAtsign"'));
```

### Steps to reproduce

Look at the code

### Expected behavior

Tests should have good logic and assertions. For example in the ttl test we could first test the positive case and then the negative, and start by setting a TTL of 6000, doing a scan, asserting the key IS returned; then set a TTL of 200, sleep for 250, do a scan, and assert that the key is NOT returned. Similar changes for ttb verifying the negative and then the positive case.

**Comment by purnimavenkatasubbu** [comment]
Fixed the assertions and raised the following [PR](https://github.com/atsign-foundation/at_server/pull/1215)

**Comment by gkc** [comment]
Will review shortly. Can you check the correctness of the other tests in this file also, please?

**Comment by purnimavenkatasubbu** [comment]
sure

**Comment by purnimavenkatasubbu** [comment]
> Will review shortly. Can you check the correctness of the other tests in this file also, please?

sure

**Comment by gkc** [comment]
Closed by #1215

**Comment by purnimavenkatasubbu** [comment]
Fixed and merged to trunk. [PR](https://github.com/atsign-foundation/at_server/pull/1215)
